pkg/visibility/server.go

```diff
@@ -90,6 +90,7 @@ func newVisibilityServerConfig() *genericapiserver.RecommendedConfig {
 	c.Config.OpenAPIV3Config.Info.Title = "Kueue visibility-server"
 	c.Config.OpenAPIConfig.Info.Version = strings.Split(c.Config.Version.String(), "-")[0]
 	c.Config.OpenAPIV3Config.Info.Version = strings.Split(c.Config.Version.String(), "-")[0]
+	c.FlowControl = nil
```

Comment:
Is it needed? Does it disable the P&F at the extension side?

Reply:
Oh, this was a different approach, removing.